docs/components/Code.tsx

```diff
@@ -75,13 +76,10 @@ function LiveCode({
 		copy(localCopy);
 	}, [localCopy]);
 
-	const handleChange = useCallback(
-		(code: string) => {
-			liveOnChange(code);
-			setLocalCopy(code);
-		},
-		[liveOnChange]
-	);
+	const handleChange = useDebouncedCallback((code: string) => {
```

Comment:
This seems to fix the deleting issue, but makes normal typing even worse… 😞 
![live-code-fix-issues](https://github.com/agriculturegovau/agds-next/assets/853552/79cece51-3847-4a62-93bb-ba5e61bd196a)

Reply:
This appears to be a bug in react-live. https://github.com/FormidableLabs/react-live/issues/343#issuecomment-1561192799

The recommendation in that thread is to switch to using [react-live-runner](https://www.npmjs.com/package/react-live-runner) instead.

https://github.com/FormidableLabs/react-live/issues/343#issuecomment-1520651394